x-pack/plugins/security_solution/public/management/cypress/tsconfig.json

```diff
@@ -22,6 +22,8 @@
       "path": "../../../tsconfig.json",
       "force": true
     },
-    "@kbn/security-plugin"
```

Comment:
This shouldn't be here...

Reply:
It is an auto-commit from kibanamachine: https://github.com/elastic/kibana/pull/148434/commits/bb552d1732e5073c0a8107c0f74145d998744bd6